docs/contributing/Building, Debugging, and Testing on Windows.md

```diff
@@ -34,7 +34,7 @@ The Test.cmd script will run our unit test on already built binaries.  It can be
 
 1. Run the "Developer Command Prompt for VS2019" from your start menu.
```

Comment:
💡 I just use Powershell now. I'm not aware of any case that needs the developer command prompt.

Reply:
The one case that I still use is simply the "I want to run MSBuild". I don't use build.cmd because every time look somebody changed what it does.